### ISSUE TITLE
Search does not work on HTML-rendered cells

### ISSUE THREAD

**Issue by jehunter5811** [grid-js/gridjs#52]
**Describe the bug**
With the new release that supports rendering html content, search does not work on cells that are wrapped in html. For example search works with this: 

```
data: documents.map(doc => [
    doc.title,
    doc.tags.map(tag => tag.name).join(', '), 
    ''
]),
```

But not with this: 

```
data: documents.map(doc => [
    html(`<button class='not-button no-underline left title-el' id=${doc.id}>${doc.title}</span>`), 
    doc.tags.map(tag => tag.name).join(', '), 
    ''
 ]),

```

**To Reproduce**
Create a cell using the html-render functionality
Try searching based on data in that cell.

**Expected behavior**
Search should work as it does when searching on non-html-rendered cells.

**Comment by afshinm** [comment]
ah, makes sense. I will take a look @jehunter5811

**Comment by afshinm** [comment]
fixed in 1.2.1

**Comment by jehunter5811** [comment]
Works great, thanks!
